Week7/app/src/main/java/com/mmoreno/pokeapp/di/ViewModelModule.kt

```diff
@@ -0,0 +1,16 @@
+package com.mmoreno.pokeapp.di
+
+import com.mmoreno.pokeapp.viewmodel.MainViewModel
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.dsl.module
+
+/**
+ * Custom Module for injecting viewModel
+ * using Koin :D
+ */
+val mainViewModule = module {
```

Comment:
Nice work and I like what you've done here passing the repository into the constructor.